tests/integration/files/file/base/win/repo-ng/7zip.sls

```diff
@@ -0,0 +1,24 @@
+{% set versions = {'18':['05', '03', '01'], '16':['04', '03', '02', '00'], '9':['20']} %}

```

Comment:
See comments

Reply:
I think this is fine as this test isn't testing the salt-winrepo repo but only the functionality of `pkg.install` just as we are also doing for `putty` in the test repo. I believe it would be more useful to add tests specifically for win-repo if we want to test the repo files themselves. Hopefully we can move towards using our own build packages as well in the future. But would like to get @twangboy and @dwoz input here as well.

Reply:
@Ch3LL If the intent is to test `pkg.install` I am fine with this. I agree adding tests specifically for the win-repo functionality is a good future step.